### ISSUE TITLE
Error cleaning up npm scenario: Error: Cannot copy '../../../../../esprima/bin/esvalidate.js' to a subdirectory of itself

### ISSUE THREAD

**Issue by elwayman02** [ember-cli/ember-try#967]
I'm seeing this error when bumping `@embroider/test-setup` from 2.1.1 to 3.0.2. It only occurs when running embroider-safe and embroider-optimized builds. Given the nature of the error seems related to #813 and is thrown from ember-try's copy functionality, this seemed the best place to file the issue for investigation.

```
Error cleaning up npm scenario: Error: Cannot copy '../../../../../esprima/bin/esvalidate.js' to a subdirectory of itself, '../../../../../esprima/bin/esvalidate.js'.
    at /home/runner/work/ember-resize-modifier/ember-resize-modifier/node_modules/ember-try/node_modules/fs-extra/lib/copy/copy.js:210:21
    at FSReqCallback.oncomplete (fs.js:180:23)
```

Here's a full PR reproduction: https://github.com/elwayman02/ember-resize-modifier/pull/808

**Comment by kategengler** [comment]
This is fixed in v3.0 of ember-try by #912

**Comment by elwayman02** [comment]
Ah, I think dependabot was having a stroke here...my library is already using ember-try but for some reason it wasn't rebasing. Sorry for the false alarm! Should be able to close this out momentarily.
